test/routes/actionRules.test.js

```diff
@@ -0,0 +1,77 @@
+describe('Action rules route test', () => {
+  let createServer
+  let server
+
+  const goodRequest = {
+    method: 'PUT',
+    url: '/actions/FG1/rules/3',
+    payload: { enabled: true }
+  }
+
+  const badRequestParams = {
+    method: 'PUT',
+    url: '/actions/FG1/rules/ABC',
+    payload: { enabled: true }
+  }
+
+  const badRequestPayload = {
+    method: 'PUT',
+    url: '/actions/FG1/rules/3',
+    payload: { enabled: '123' }
+  }
+
+  const mockUpdatedRule = {
+    id: 3,
+    description: 'Rule description',
+    enabled: false,
+    facts: [{ description: 'Total Parcel Perimeter' }]
+  }
+
+  const mockActionRulesService = {
+    update: jest.fn().mockResolvedValue(mockUpdatedRule)
+  }
+
+  beforeAll(async () => {
+    jest.mock('../../server/services/actionRulesService', () => mockActionRulesService)
+    createServer = require('../../server/createServer')
+  })
+
+  beforeEach(async () => {
+    server = await createServer()
+    await server.initialize()
+  })
+
+  test('calls the actionRulesService', async () => {
+    await server.inject(goodRequest)
+    expect(mockActionRulesService.update).toHaveBeenCalled()
```

Comment:
I think we should test that the correct parameters were passed to `mockActionRulesService.update`. 
Something like this?
```
expect(mockActionRulesService.update).toHaveBeenCalledWith(actionID, ruleID, goodRequest.payload.enabled)
```

Reply:
Test has been updated.